app/views/compute_resources_vms/form/vmware/_base.html.erb

```diff
@@ -18,6 +18,12 @@
 <% checked = params[:host] && params[:host][:compute_attributes] && params[:host][:compute_attributes][:start] || '1' %>
 <%= checkbox_f f, :start, { :checked => (checked == '1'), :help_inline => _("Power ON this machine"), :label => _('Start'), :label_size => "col-md-3" } if new_host and controller_name != 'compute_attributes' %>
 
+<% checked = params[:host] && params[:host][:compute_attributes] && params[:host][:compute_attributes][:memoryHotAddEnabled] || '0' %>
+<%= checkbox_f f, :memoryHotAddEnabled, { :checked => (checked == '1'), :help_inline => _("Enable memory hot add"), :label => _('Memory hot add'), :label_size => "col-md-2" } if new_host and controller_name != 'compute_attributes' %>
```

Comment:
I don't know if you want the `and controller_name != 'compute_attributes'` here, as that's to prevent the checkbox appearing on the compute profile pages.  It's excluded from there for the "Power ON" checkbox as it's kind of only applicable to real hosts, not compute profiles, but I think these would be useful set on compute profiles too.

I think that means you can also get rid of the "checked" variable and simply leave :checked out, a bit like external_ip on in gce/_base.